src/components/LoginForm/FormUser.tsx

```diff
@@ -0,0 +1,79 @@
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useMutation } from 'react-query';
+import { AxiosResponse } from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { Button, ForgetPassword, Form, InputForm, TitleLogin } from './Login.styled';
+import loginSchema from '../../utils';
+
+import { authRoutes } from '../../services';
+
+interface FormData {
+  email: string;
+  password: string;
+}
+
+interface ReqError extends AxiosResponse {
+  response: {
+    data: {
+      error: boolean;
+      data: {
+        message: string;
+      };
+    };
+  };
+}
+
+const LoginForm = () => {
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormData>({ resolver: zodResolver(loginSchema), mode: 'onBlur' });
+  const { isLoading, mutate, error } = useMutation((data: FormData) => authRoutes.login(data), {
+    onSuccess: (data) => {
+      localStorage.setItem('user', JSON.stringify(data.data.data.user));
+
+      // eslint-disable-next-line no-unused-expressions
+      data.data.data.user ? navigate('/academy') : '';
```

Comment:
What if the admin trying to log in from this page?

e.g.:
```ts
if (data.data.data.user.isAdmin) {
  navigate('/admin');
} else {
  toast.error('This page just for Admin');
}
```

Reply:
I'm sorry, I think I copied and pasted the wrong code.
What I want to tell you is to handle the error if the admin is trying to log in from the user login page

```ts
if (data.data.data.user.isAdmin) {
  toast.error('Try to login from admin page please ');
} else {
  t// implement you login for user
}
```

Reply:
done